src/components/timers/Countdown.tsx

```diff
@@ -0,0 +1,189 @@
+import Button from '../generic/Button';
+import DisplayTime from '../generic/DisplayTime';
+import Inputs from '../generic/Inputs';
+import Panel from '../generic/Panel';
+
+import { useEffect, useRef, useState } from 'react';
+import { getDisplayHundredths, getDisplayMinutes, getDisplaySeconds } from '../../utils/helpers';
+
+interface CountdownProps {
```

Comment:
Your individual timer components have a core  shared interface (with some having a couple of extra props). 

You can DRY this up by creating a base interface and then extending it where needed:
https://www.typescriptlang.org/docs/handbook/2/objects.html#extending-types